examples/01_forecasting.ipynb

```diff
@@ -60,7 +60,8 @@
     "        * [1.2.1 Basic deployment workflow in a nutshell](#section_1_2_1)\n",
     "        * [1.2.2 Forecasters that require the horizon already in `fit`](#section_1_2_2)\n",
     "        * [1.2.3 Forecasters that can make use of exogeneous data](#section_1_2_3)\n",
-    "        * [1.2.4 Prediction intervals](#section_1_2_4)      \n",
+    "        * [1.2.4 Multivariate Forecasters](#section_1_2_4)\n",
+    "        * [1.2.5 Prediction intervals](#section_1_2_5)      \n",
```

Comment:
I would also include quantile forecasts, i.e., "prediction intervals and quantile forecasts"